website/server/models/group.js

```diff
@@ -504,6 +504,7 @@ function setUserStyles (newMessage, user) {
       spookySparkles: userCopy.stats.buffs.spookySparkles,
       snowball: userCopy.stats.buffs.snowball,
     };
+    userStyles.stats.class = userCopy.stats.class;
```

Comment:
LGTM 👍 , but I think this should be moved outside of the `if`-statement, incase there isn't a `buffs` property

Reply:
thanks for reviewing @negue !
I've moved the `stats.class` assignment outside the `if (buffs)` statement